### DISCUSSION TITLE
结合 Gitpod 做PR预览是否可以？

### DISCUSSION THREAD

**Discussion by giscafer** [LeekHub/leek-fund#305] [Ideas]
## Gitpod
https://www.gitpod.io/

## 之前网友的截图

![](https://user-images.githubusercontent.com/6399899/102346139-5a473000-3fd9-11eb-9608-58566f6857f6.png)

**Comment by James-chok** [comment]
可以试一下，我原来还期望vscode online可以调试运行

**Reply by giscafer** [reply]
gitpod 的话可以直接拉pr的最新代码，在线运行插件的话，就相当于可以实现预览了。也是可以调试的

**Reply by James-chok** [reply]
直接运行代码，需要考虑限制代码更改或代码提交

**Reply by giscafer** [reply]
开个只读的token 应该就可以吧
